src/commands/info/leaderboard.ts

```diff
@@ -102,7 +102,7 @@ export default class extends Command {
 			.setDescription(content)
 			.setImage('attachment://leaderboard.png');
 
-		interaction.editReply({
+		interaction.followUp({
```

Comment:
I know this works, I was just wondering why you wanted to followUp instead of editing the reply?

Reply:
I believe Ben glossed over it in #241 but it is because sometimes it takes longer than 3 seconds (the usual time Discord waits before it spits out `this interaction failed`) to generate the leaderboard image. This function guarantees that the user will receive a proper reply, even if it takes >3 sec. `followUp()` must be used in tandem with `deferReply()`.

Reply:
example - https://discord.com/channels/744982373300437627/745744955674329168/939870597071130704